sdk/pinocchio/src/program.rs

```diff
@@ -189,21 +183,15 @@ pub unsafe fn invoke_unchecked(instruction: &Instruction, accounts: &[Account])
 /// Invoke a cross-program instruction with signatures but don't enforce Rust's
 /// aliasing rules.
 ///
-/// This function does not check that [`Ref`]s within [`Account`]s are properly
-/// borrowable as described in the documentation for that function. Those checks
-/// consume CPU cycles that this function avoids.
+/// This function does not check that [`Account`]s are properly borrowable.
+/// Those checks consume CPU cycles that this function avoids.
 ///
 /// # Safety
 ///
 /// If any of the writable accounts passed to the callee contain data that is
 /// borrowed within the calling program, and that data is written to by the
 /// callee, then Rust's aliasing rules will be violated and cause undefined
 /// behavior.
-///
-/// # Important
-///
-/// The accounts on the `account_infos` slice must be in the same order as the
-/// `accounts` field of the `instruction`.
```

Comment:
Same here, this is pretty important! You can also add it under the "Safety" section if you'd like

Reply:
Same here, `*_unchecked` version does not do this validation so the order is not important.

Reply:
Ah ok, thanks!